server/spinwick.go

```diff
@@ -459,7 +461,7 @@ func (s *Server) createSpinWick(pr *model.PullRequest, size string, withLicense
 		return request.WithError(errors.Wrap(request.Error, "error waiting for installation to become stable")).ShouldReportError()
 	}
 
-	spinwickURL := fmt.Sprintf("https://%s.%s", s.makeSpinWickID(pr.RepoName, pr.Number), s.Config.DNSNameTestServer)
+	spinwickURL := fmt.Sprintf("https://%s.%s", s.makeUniqueSpinWickID(pr.RepoName, pr.Number), s.Config.DNSNameTestServer)
```

Comment:
Here, we should probably use the `spinwickID` we created above. Maybe we could go one step further and initialize a common `spinwickDNS` variable since most of the formatting is the same.

Reply:
I like this idea, and I have another thought to build on your suggestion further - will re request review when I've implemented